<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Apply a display filter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTreeView" name="detailTreeView"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="placeholderText">
       <string>Choose a Adapter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTableView" name="packetTableView"/>
    </item>
    <item row="4" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QTextEdit" name="acsiiTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSniffer">
    <property name="title">
     <string>Sniffer</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
   </widget>
   <addaction name="menuSniffer"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actioncapture">
   <property name="text">
    <string>capture</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
